pkg/controller/migmigration/backup.go

```diff
@@ -5,7 +5,8 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```

Comment:
@jortel I'm lost as to why this got auto-aliased, I did not make this change. Any ideas?

Reply:
No idea.